crates/weaver_resolved_schema/src/body.rs

```diff
@@ -0,0 +1,91 @@
+// SPDX-License-Identifier: Apache-2.0
+
+#![allow(rustdoc::invalid_html_tags)]
+
+//! Specification of a resolved body field.
+
+use crate::attribute::AttributeRef;
+use crate::value::Value;
+use schemars::JsonSchema;
+use serde::{Deserialize, Serialize};
+use weaver_semconv::body::{BodySpec, BodyFieldSpec};
+use weaver_semconv::attribute::{AttributeType, Examples, RequirementLevel};
+use weaver_semconv::stability::Stability;
+
+/// An attribute definition.
+#[derive(Serialize, Deserialize, Debug, Clone, Eq, PartialEq, Hash, JsonSchema)]
+#[serde(deny_unknown_fields)]
+pub struct Body {
+    /// The body specification.
+    #[serde(skip_serializing_if = "Option::is_none")]
+    pub fields: Option<Vec<BodyField>>,
+
+    /// The body value when there are no fields
+    #[serde(skip_serializing_if = "Option::is_none")]
+    pub value: Option<Value>
```

Comment:
Added this show how a simple value might be implemented, I believe I currently have a panic in place if this is used as I introduced this to ensure that what I've created can be extended without too much difficulty -- at least that was the theory

Reply:
See https://github.com/open-telemetry/semantic-conventions/issues/755 for why I added this, basically I'm trying to setup how the body (for logs) would be modelled in Sem conv and while we have only currently agreed to support a single level map (the fields) at some point we will need to support the Logs "Any" value

Reply:
I'm going to comment these out for now that I've got the basic support until we agree on the exact definition that we want to move forward with (from the issue link 755 above)